frontend/packages/core/src/Assets/icons/ExperimentIcon.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from "react";
+
+import type { SVGProps } from "../global";
+
+import StyledSvg from "./helpers";
+
+const ExperimentIcon = (props: SVGProps) => (
+  <StyledSvg
+    width="24"
+    height="24"
```

Comment:
You can remove the height and width since it's set by the SVGProps `size` prop

Reply:
when I do that they get massive. I tried a bunch of things like passing various sizes in ("small", "medium", "large" etc) removing / adding the viewbox, and setting width and height from props and none of it changes. I also tried using `StyledSVG` instead of `StyledSvg` so that I could pass the size in but that doesn't work either.

I see in the `SVGProps` in global.ts it sets the default to be small (aka 24) but its not getting set on the icons. Either I'm doing something wrong or there is some issue with the current SVGProps that makes it not work for this case.
![Screen Shot 2022-05-11 at 5 42 19 PM](https://user-images.githubusercontent.com/66325812/167970302-c2431f59-73fc-4160-aa12-80b8c9ef10d3.png)
![Screen Shot 2022-05-11 at 5 42 07 PM](https://user-images.githubusercontent.com/66325812/167970303-58062ffc-89b4-4f0a-b395-8362e9865dfd.png)

Reply:
Ok let me take a deeper look today

Reply:
So looking into this, the issue is that `StyledSvg` in the icons is coming from `helpers.tsx` which is not the same as the `StyledSVG` from the `global.ts` file. Modifying that and changing the icon to extend the size it works as expected.

```diff
--- a/frontend/packages/core/src/Assets/icons/ExperimentIcon.tsx
+++ b/frontend/packages/core/src/Assets/icons/ExperimentIcon.tsx
@@ -1,17 +1,14 @@
 import * as React from "react";

 import type { SVGProps } from "../global";
+import { StyledSVG } from "../global";

-import StyledSvg from "./helpers";
-
-const ExperimentIcon = (props: SVGProps) => (
-  <StyledSvg
-    width="24"
-    height="24"
+const ExperimentIcon = ({ size, ...props }: SVGProps) => (
+  <StyledSVG
+    size={size}
     viewBox="0 0 24 24"
     fill="none"
     xmlns="http://www.w3.org/2000/svg"
-    hoverFill="#00809D"
     {...props}
   >
     <path
@@ -20,7 +17,7 @@ const ExperimentIcon = (props: SVGProps) => (
       fillRule="evenodd"
       clipRule="evenodd"
     />
-  </StyledSvg>
+  </StyledSVG>
 );
```